### DISCUSSION TITLE
How do I enable Banners?

### DISCUSSION THREAD

**Discussion by SakakiIsLove** [vichan-devel/vichan#719] [Q&A]
I have setup the board I intend to use but there are no banners showing, i have afew banners in my inc/static/banners folder but none of them display.

![image](https://github.com/vichan-devel/vichan/assets/166055543/a595dbe4-a32d-4407-8896-89d488963738)

**Comment by tinfoil-hat-net** [comment]
Maybe `chown www-data:www-data /path/to/image` helps

**Comment by chanadmins** [comment]
go to inc/config.php and enable banners
